lib/private/User/User.php

```diff
@@ -89,6 +89,9 @@ class User implements IUser {
 	/** @var int|null */
 	private $lastLogin;
 
+	/** @var int|null */
+	private $firstLogin;
```

Comment:
```suggestion
	private ?int $firstLogin;
```
Please strong type new code